stack/Stack.java

```diff
@@ -0,0 +1,88 @@
+/*
+*  Following implementation of Stack uses LinkedList
+*  The last element of LinkedList is considered as the Top of Stack
+*  T defines the type of Stack we wish to create
+*/
+
+import java.util.LinkedList;
+import java.util.NoSuchElementException;
+
+class Stack<T> {
+    private int size;
+    private LinkedList<T> stack;
+        
+    public Stack() {   // Constructor to create empty Stack
+        stack = new LinkedList<> ();
+        size = 0;
+    }
+
+  	public void push(T data) { // Add element to Top of Stack
+        stack.addLast(data);
+        size++;
+    }
+        
+    public T pop() {    // Remove element from top of Stack
+        T temp = null;
+        try {
+            temp = stack.removeLast();
+            size--;
+        }
+        catch (NoSuchElementException exp) {
```

Comment:
So, you are saying that I just throw the exception from methods and introduce a try-catch in main so that it will catch exception there and thus leading them to know what kind of exception is occurring.